akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -368,7 +368,10 @@ private[akka] class LocalActorRef private[akka] (
    */
   def getSingleChild(name: String): InternalActorRef = actorCell.getSingleChild(name)
 
+  lazy val log = Logging(_system, getClass)
+
   override def getChild(names: Iterator[String]): InternalActorRef = {
+    log.warning("=== getChild: " + names.mkString("/"))
```

Comment:
leaving all those debug things in for now.

Reply:
This log statement breaks everything -- gotta love `Iterator` </3 Thanks @jrudolph for helping to spot this